Quantum-Control-Applications/Superconducting/Single Fixed Transmon/configuration.py

```diff
@@ -64,7 +64,6 @@ def readout_macro(threshold=None, state=None, I=None, Q=None):
 #############
```

Comment:
For the future when we'll have more of them, I think it is better to put the QUA macros in a separated file called macros.py rather than in the configuration

Reply:
Yea I agree.
But not for this PR